src/NServiceBus.Transport.RabbitMQ/Timeouts.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace NServiceBus
+{
+    /// <summary>
+    /// Described the support of timeouts when running RabbitMQ transport in a RabbitMQ cluster.
+    /// </summary>
+    public enum Timeouts
```

Comment:
```suggestion
    public enum DelayedDelivery
```

To align with https://docs.particular.net/nservicebus/messaging/delayed-delivery ?

Reply:
makes sense